pkgs/development/libraries/glibc/fix_path_attribute_in_getconf.patch

```diff
@@ -3,4 +3,4 @@ diff -ubr glibc-2.17-orig/sysdeps/unix/confstr.h glibc-2.17/sysdeps/unix/confstr
 +++ glibc-2.17/sysdeps/unix/confstr.h	2013-06-03 22:04:39.469376740 +0200
 @@ -1 +1 @@
 -#define	CS_PATH	"/bin:/usr/bin"
-+#define	CS_PATH	"/run/current-system/sw/bin:/bin:/usr/bin"
++#define	CS_PATH	"/run/current-system/sw/bin:/nix/var/nix/profiles/default/bin"
```

Comment:
```suggestion
+#define	CS_PATH	"/run/current-system/sw/bin:/nix/var/nix/profiles/default/bin:/usr/bin:/bin"
```

Per the other comments, we want to keep `/bin` and `/usr/bin` so that binaries work on linux systems other than NixOS. They do no harm while running on NixOS